src/components/Section/stories/Section.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import Section from "../Section";
+
+const meta: Meta<typeof Section> = {
+  title: "Components/Section",
+  component: Section,
+};
+
+export default meta;
+type Story = StoryObj<typeof Section>;
+
+export const Default: Story = {
+  args: {
+    children: (
+      <>
+        <h2>I am a title</h2>
+        <p>And I am a paragraph</p>
+      </>
+    ),
+    theme: "Light",
+  },
+};
```

Comment:
Si tu veux des manière plus sympa que de passer la props children tu peux faire comme ça : https://storybook.js.org/docs/writing-stories/stories-for-multiple-components#creating-a-template-component